tiny/lisp-nontail.c

```diff
@@ -18,16 +18,24 @@
   Public Domain
 */
 
-#include <stdlib.h>
-#include <stdio.h>
-#include <string.h>
+#include <ctype.h>
+#include <inttypes.h>
 #include <setjmp.h>
 #include <stdarg.h>
-#include <ctype.h>
-#include <sys/types.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
 
-typedef u_int32_t value_t;
+#ifdef __LP64__
```

Comment:
Why not `uintptr_t` and `intptr_t`?

Reply:
You're right. If we use `stdint.h` (a decision I agree with) we should use those types. The original FL code used the custom (BSD-derived?) type `u_int32_t` so I went with the symmetrical 64-bit type in the PR.